gRPC_test/coffeeServiceClient/coffeeService.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
```

Comment:
This is a copy of gRPC_test/src/main/proto/coffeeService.proto? Keep it only once